modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -3,28 +3,31 @@
 // * logging/cluster-logging-deploying.adoc
 
 [id="cluster-logging-deploy-multitenant_{context}"]
-= Installing OpenShift Logging into a multitenant network
+= Installing OpenShift Logging with SDN network isolation enabled
 
-If you are deploying OpenShift Logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+In a cluster, if the xref:../networking/openshift_sdn/about-openshift-sdn.adoc#nw-openshift-sdn-modes_about-openshift-sdn[OpenShift SDN network isolation mode] uses the _Network policy_ or _multitenant_ modes, projects in that cluster are isolated from each other. Both of these modes prevent network traffic between pods or services that are in different projects. Network policy is the default mode in {product-title} {product-version}.
 
-Because the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator are installed in different projects, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. How you allow this access depends on how you configured multitenant isolation mode.
+When you deploy OpenShift Logging, the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator are installed in different projects. Therefore, if a cluster is configured to use the Network policy or multitenant modes, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. The method you use to allow access depends on the network isolation mode.
 
 .Procedure
 
-To allow traffic between the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator, perform one of the following:
+To allow traffic between the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator, perform either of the following actions:
 
-* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, use the following command to join the two projects:
-+
-For example:
+* If you configured network isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, join the two projects. For example:
 +
 [source,terminal]
 ----
 $ oc adm pod-network join-projects --to=openshift-operators-redhat openshift-logging
 ----
 
-* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *NetworkPolicy* mode, create a network policy object in the `openshift-logging` namespace that allows ingress from the `openshift-operators-redhat` project to the `openshift-logging` project.
+* If you configured network isolation mode with the OpenShift SDN CNI plug-in set to the *NetworkPolicy* mode, set a label on the `openshift-operators-redhat` namespace. For example:
```

Comment:
Or this _if_ statement could also be _a_ and _b_ steps: Set the namespace label, and then create & apply the network policy object in the correct namespace. It really is two+ discrete steps.